regression/tests/enterprise/test_percentage_discount_ent_offer.py

```diff
@@ -0,0 +1,133 @@
+"""
+Enterprise Offer Discount tests
+"""
+from regression.tests.enterprise.ent_test_base import EnterpriseTestBase
+from regression.pages.whitelabel.basket_page import SingleSeatBasketPage
+
+
+class TestDiscountEnterpriseOffer(EnterpriseTestBase):
+    """
+    Tests for Percentage Discount Enterprise Offers
+    """
+    DISCOUNT_MSG = "Discount provided by "
+
+    def setUp(self):
+        super(TestDiscountEnterpriseOffer, self).setUp()
+
+    def test_percentage_offer_detail_landing_page(self):
+        """
+        Scenario: To verify that user sees the correct discount
+        percentage info and detail on enterprise landing page
+        Given a user has an edx account which is linked to an Enterprise
+        portal account When this user lands on the enrollment landing page
+        Then this user is shown correct discount details
+        """
+        self.register_and_go_to_course_enrollment_page()
+        # Call the fixture to unlink existing account for the user
+        self.addCleanup(self.unlink_account)
+        self.assertEqual(
+            self.ENT_COURSE_TITLE,
+            self.ent_course_enrollment.get_course_title()
+        )
+        # Verify that course type "verified" is present as a selectable option
+        self.assertTrue(
+            self.ent_course_enrollment.target_course_type_is_present(
+                "verified"
+            )
+        )
+        self.assertIn(
+            self.ENT_COURSE_PRICE,
+            self.ent_course_enrollment.get_course_discounted_price()
+        )
+        self.assertIn(
+            self.ENT_COURSE_DISCOUNTED_PRICE,
+            self.ent_course_enrollment.get_course_discounted_price()
```

Comment:
why are both course price and discounted prices both verified against one function
and why are you using assertin instead of assertEqual?